app/move/fields/update.js

```diff
@@ -0,0 +1,24 @@
+const { cloneDeep } = require('lodash')
+
+const createFields = require('./create')
+
+const updateFields = cloneDeep(createFields)
+
+updateFields.police_national_computer = {
+  ...updateFields.police_national_computer,
+  readOnly: true,
+  updateComponent: {
+    component: 'appReadOnlyField',
+    classes: '',
+    items: [
+      {
+        component: 'govukDetails',
+        summaryHtml: 'I need to change the PNC number',
```

Comment:
Should these be in the locales files?

Reply:
They totally should.

Reply:
Moved. And updated `common/helpers/field.js` to handle the new property